content/docs/contributing/to-code/git-workflow.md

```diff
@@ -22,7 +22,7 @@ At a high level, the workflow is:
 
 3. If there are no merge conflicts, automated review (AR) is triggered and the pull request is flagged for code review.
 
-4. If the pull requests passes code review, the SIG owner (or a delegate) will merge it into the `main` branch of the O3DE GitHub repo.
+4. If the pull requests passes code review, the SIG maintainer (or a delegate) will merge it into the `main` branch of the O3DE GitHub repo.
```

Comment:
Should this be changed to `development`?

Reply:
Good catch. I'll update it to dev.